utils/index.ts

```diff
@@ -126,11 +133,56 @@ async function deleteCurrentDeviceShare(page: Page) {
   }
 }
 
+function randomString(length: number) {
+  var chars = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  var result = "";
+  for (var i = length; i > 0; --i)
+    result += chars[Math.floor(Math.random() * chars.length)];
+  return result;
+}
+
+function randomNumber() {
+  return Math.floor(Math.random() * 10);
+}
+
+async function setup2FA(page: Page, flow: string) {
```

Comment:
If we put this into utils, we expect that this function can be reused in multiple places.